lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -98,16 +99,21 @@ def instructor_dashboard_2(request, course_id):
 
     is_white_label = CourseMode.is_white_label(course_key)
 
-    sections = [
-        _section_course_info(course, access),
-        _section_membership(course, access, is_white_label),
-        _section_cohort_management(course, access),
-        _section_student_admin(course, access),
-        _section_data_download(course, access),
-    ]
+    sections = []
+    available_tabs = _available_instructor_tabs(request)
```

Comment:
If `_available_instructor_tabs` function is used only once you may push code block without it